services/web-app/components/svg-libraries/svg-library.module.scss

```
@@ -9,7 +9,7 @@
 .svg-page {
   // prevent jank while importing icons
   min-height: 60vh;
-  padding-top: spacing.$spacing-11 + spacing.$spacing-05;
+  padding-top: spacing.$spacing-05;
```

Still see a small padding difference with carbondesignsystem.com, I think it should be spacing-07 here
```suggestion
  padding-top: spacing.$spacing-07;
```